インデックス: my_djangos/templates/guestboard/index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- my_djangos/templates/guestboard/index.html	(date 1579412782004)
+++ my_djangos/templates/guestboard/index.html	(date 1579412782004)
@@ -16,7 +16,7 @@
                     {% endif %}
                     <div class="col-sm-10 text_change_color">
                         <label class="col-sm-3 control-label ">{{ field.label_tag }}</label>
-                        <label class="label col-sm-7 text_change_color">{{ field }}</label>
+                        <label class="label col-sm-7 ">{{ field }}</label>
                     </div>
                 </div>
             {% endfor %}
